_docs-sources/pipelines/security/audit-log.md

```diff
@@ -0,0 +1,116 @@
+# Audit logs
+
+Gruntwork Pipelines provides an audit log of which user performed what action in which account. To accomplish this, Pipelines sets the AWS STS session name using a combination of the initiating GitHub user, the name of pipelines itself, and the pull request or branch from which the action was triggered. All log data for Gruntwork Pipelines is done using [AWS CloudTrail](https://aws.amazon.com/cloudtrail/). Session names are used in the `User name` field in CloudTrail, allowing those searching the data to clearly identify which user performed an action. For more information on querying the logs see [where you can find logs](#where-you-can-find-logs) and [querying data](#querying-data).
+
+### Who gets logged
+
+Pipelines uses a naming scheme that combines the GitHub user who triggered the [action](../overview/actions.md) and the pull request or branch from which the action was initiated. There are two scenarios in which Pipelines runs actions - when a pull request is opened, synchronized, or re-opened, and when a pull request is merged to the deploy branch (e.g., `main`).
```

Comment:
We should call "actions" "Pipelines Actions" so that we disambiguate them from "GitHub Actions." 

Separately, it might be worth starting this section with the "formula" for the session name, so something like:

_Pipelines will set the `user name` property of the AWS STS session according to the following format: `<$GitHubUserName-via-GWPipelines@PR-$PullRequestNumber>`._

You could then explicitly describe how the `$GitHubUserName` portion is set, clarifying that it's based on whatever GitHub user took an action that triggered the GitHub Action running Pipelines to run.

Reply:
Stylistic preference: We should use the colon instead of the dash to introduce an idea. For example:

_There are two scenarios in which Pipelines runs Pipelines Actions: when a pull request is opened, updated, or re-opened; and when a pull request is merged to the deploy branch (e.g., main)._

For this sentence in particular, you could simplify it even further to:

_Pipelines runs Pipelines Actions when a pull request is opened, updated, re-opened, or merged to the deploy branch (e.g., `main`)._

Reply:
Addressed in [e3f2fb2](https://github.com/gruntwork-io/docs/pull/1143/commits/e3f2fb2f2d7ff2fa02709cd3bd6cd2242c25f5be)